warehouse/templates/manage/manage_base.html

```diff
@@ -137,7 +186,20 @@ <h3 class="modal__title">{{ title }}{% if confirm_string_in_title %} {{ confirm_
   <a href="#{{ slug }}" class="button button--danger">
     {{ title }}
   </a>
-  {{ confirm_modal(title, confirm_name, confirm_string, slug, index=None, extra_fields=extra_fields, action=action) }}
+  {{ confirm_modal(title, confirm_name, confirm_string, slug, extra_fields=extra_fields, action=action) }}
+{% endmacro %}
+
+{% macro confirm_password_button(
+  title,
+  extra_fields=None,
+  action=None
+)
+%}
+  {% set slug = title.lower().replace(' ', '-') + '-modal' + ('-{}'.format(index) if index else '') %}
```

Comment:
It seems like there will never be an `index` here?

Reply:
Yeah, good catch. The reason I went for removing it is because none of the calls to `confirm_button` actually use it, they simply the title which seems to be enough. I'll reinstate it anyway.

Reply:
Agh, got it the other way around 😄 I'll remove it